quantities/tests/test_umath.py

```diff
@@ -154,17 +154,13 @@ def test_ceil(self):
             [-1., -1., -0., 1., 2., 2., 2.] * pq.m
             )
 
-    @unittest.expectedFailure
```

Comment:
Why was this change and the change below needed? It doesn't seem to be related to the bug fix in the tolist mehtod.

Reply:
@thisch The numpy `.fix` method is something that we would like to behave the same in numpy and in quantities (i.e. it should round towards zero).  It used to not work with quantities, which is why the expected failure was there.  But it works now, because of changes in numpy I think (though it could be the result of my changes)?  So I removed the expected failure because it now passes as it should.